### ISSUE TITLE
Preset speed drive birddog p200?

### ISSUE THREAD

**Issue by rrook97** [bitfocus/companion#1255]
Hello everyone,

I have a question about the birddog p200 camera.

we want to be able to adjust the preset speed via companion. but this does not work, anyone have an idea?

**Comment by josephdadams** [comment]
Which module are you using?

**Comment by rrook97** [comment]
by module you mean the companion module? i have the sony visca or the ptz optics.

**Comment by josephdadams** [comment]
Yes, which companion module are you using to control your birddog p200? I am going to transfer this issue to that module repository instead of the core Companion repository, which is used for core Companion bug issues and feature requests.

**Comment by rrook97** [comment]
we use the Sony VISCA module from companion
